lib/inventory_refresh/logging.rb

```diff
@@ -4,7 +4,13 @@ class << self
   end
 
   def self.logger
-    @logger ||= NullLogger.new
+    @logger ||= default_logger
+  end
+
+  def self.default_logger
+    logger = Logger.new(STDOUT)
+    logger.level = Logger::ERROR
+    logger
```

Comment:
Is this related to mark&sweep?  If you want to pass in a logger for tests lets do that not change the default logger.

Reply:
well I'd like to see at least ERRs as default, that is for specs and also for dev, otherwise it's confusing not to see errors anywhere :-)

not sure if we can make it default per ENV, still don't have anything like Rails.env, right?

Reply:
I don't expect a gem I include to start writing to STDOUT

Reply:
so what would be the place to put it on, so it works as default for the gems specs and for dev?

Reply:
You could set the logger in the spec_helper and bin/console for dev.  For any apps including the gem we already set the logger or should if we don't

Reply:
ok, will do